src/integrators/batchNormNN.jl

```diff
@@ -77,8 +77,13 @@ function apply(this::batchNormNN{T},theta::Array{T},Y::Array{T,2},tmp::Array,doD
 
         Yd::Array{T,2}, Y, tmp[i,2] = apply(this.layers[i],theta[cnt+(1:ni)],Y,doDerivative)
         if this.outTimes[i]==1
-            Ydata = [Ydata; this.Q*Yd]
+            if typeof(this.Q) <: UniformScaling
+                Ydata = Yd # no-op
```

Comment:
Maybe should be Ydata = [Ydata ; Yd] In case both affine and norm layer are outtimes=1 !?